src/index.css

```diff
@@ -1,13 +1 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+@import "~antd/dist/antd.css";
```

Comment:
this imports the CSS for the entire antd library, which is 60kb. Ideally we will only import the css required by the components we use. The ant docs talk about ways around this in the [advanced guide](https://ant.design/docs/react/use-with-create-react-app#Advanced-Guides). I'm cool with this being merged for now as long as a ticket is created and prioritized next sprint.

Reply:
https://jira.mongodb.org/browse/MCI-3156